### PR TITLE
Adds unit test case for slack util script

### PR DESCRIPTION
**Pull request by bhuvangupta008**
Adding unit test for Slack util file. Much of it is same as lib util file

### CODE REVIEW

File: lib/slack/util.py [hunk 1]
```diff
@@ -3,9 +3,11 @@
 import igraph
 import lib.slack.config as config
 import pickle
+
```
Comment (prasadtalasila) [inline lib/slack/util.py]: The global variable needs to be removed.

File: lib/slack/util.py [hunk 2]
```diff
@@ -86,7 +91,8 @@ def get_year_month_day(day_content):
         Returns:
             str:year, str:month, str:day
     """
-    year, month, day = str(day_content["auxiliary_data"]["year"]), str(day_content["auxiliary_data"]["month"]), str(day_content["auxiliary_data"]["day"])
+    year, month, day = str(day_content["auxiliary_data"]["year"]), str(day_content["auxiliary_data"]["month"]), str(
+        day_content["auxiliary_data"]["day"])
```
Comment (prasadtalasila) [inline lib/slack/util.py]: Two more tabs would align the right side of the assignment statement.

**Review by prasadtalasila — CHANGES_REQUESTED** [review]

File: lib/slack/util.py (hunk 2 above)
Comment (bhuvangupta008) [reply lib/slack/util.py]: done

File: lib/slack/util.py (hunk 1 above)
Comment (bhuvangupta008) [reply lib/slack/util.py]: was unused. removed it

**Review by prasadtalasila — APPROVED** [review]

**Pull request merged**